tools/authors/search.inc

```diff
@@ -173,7 +173,10 @@ function get_search_title() {
 function search() {
     global $sortUtility, $sql_what_to_list, $search_result;
     $orderby = $sortUtility->getOrderBy();
-    $search_result = mysqli_query(DPDatabase::get_connection(), "SELECT * FROM authors WHERE $sql_what_to_list ORDER BY $orderby");
```

Comment:
Leave the two variables in the string directly, but do break it out into an `$sql=`. Also, remove the `false` from query().